brainz-mmd2-jaxb/src/main/java/org/musicbrainz/mmd2/AliasList.java

```diff
@@ -75,10 +74,12 @@ public class AliasList {
      * {@link Alias }
      * 
      * 
+     * @return
+     *     The value of the alias property.
      */
     public List<Alias> getAlias() {
         if (alias == null) {
-            alias = new ArrayList<Alias>();
+            alias = new ArrayList<>();
```

Comment:
Is dropping `Alias` here intentional and correct?

Reply:
Yes

Reply:
[Type can be infered since Java SE 7](https://dev.java/learn/generics/intro/#the-diamond), but JAXB runtime kept specifying the type in **generated code,** most probably for previously supported targets (earlier Java SE versions).